src/server/config/sitemap.js

```diff
@@ -30,6 +30,12 @@ export default (server) => {
           changefreq: 'weekly',
           priority: 1
         });
+
+        urls.push({
+          url: `/${chapter.chapterNumber}/info`,
```

Comment:
Keys conversion isn't happening here, not sure why. Switch to `chapter.chapter_number` or `chapter.id`